packages/cli/package.json

```diff
@@ -27,7 +27,8 @@
     "format": "prettier --write --config .prettierrc.json --ignore-path .prettierignore \"{src,tests}/**/*.ts\" \"*.{js,json}\"",
     "lint:fix": "eslint \"src/**/*.ts\" \"tests/**/*.ts\" --fix",
     "lint": "eslint \"src/**/*.ts\" \"tests/**/*.ts\"",
-    "precommit": "lint-staged"
+    "precommit": "lint-staged",
+    "format-check": "bash ../../.github/scripts/lint-pr.sh $npm_config_source 14"
```

Comment:
maybe it's better not to depends on external files

Reply:
it's better to extract the bash script as a common file for easy understanding or fix

Reply:
Follow-up with eslint check, you can also write the code into the common file

Reply:
thank you! 
code updated.